grpc/test_support/src/lib.rs

```diff
@@ -92,7 +92,7 @@ pub type Hash = [u8; 32];
 pub use crate::internal::DEFAULT_ACCOUNT_ADDR;
 
 /// Default initial balance of a test account in motes.
-pub const DEFAULT_ACCOUNT_INITIAL_BALANCE: u64 = 50_000_000_000_000u64;
+pub const DEFAULT_ACCOUNT_INITIAL_BALANCE: u64 = 500_000_000_000_000_000u64;
```

Comment:
cc @EdHastingsCasperLabs

Reply:
I think it was too large when I was trying to fix the gaslimit error again after merging master. Decreased to the amount that's present in `accounts.csv`